content/news/2017-HLSPs.md

```diff
@@ -0,0 +1,25 @@
+Title: New and Updated K2 High Level Science Products
+Date: 2017-08-16 12:00
+Author: Ann Marie Cody
+
+The K2 communinity has been very active in working with the mission's images and light curves, and a number of groups have made 
+their output available to the public in the form of High Level Science Products. We are happy to report updates to several of these 
+submissions over the last 6 months. The K2 Mission encourages further submissions of HLSPs and our Guest Observer Office is happy 
+to help astronomers interface with the MAST archive.
+
+This month, the [K2 Systematics Correction](https://archive.stsci.edu/prepds/k2sc/) ("K2SC") by Aigrain et al. is celebrating the 
+release of version 2, with light curves now available through Campaign XX. These data products provide time series detrended for 
+positional jitter and/or time-dependent systematics. As such, they are well suited to studying variable stars. 
+
+In addition, as of July 26, the [K2SFF](https://archive.stsci.edu/prepds/k2sff/) data produced by Vanderburg & Johnson now include 
+Campaign 11 light curves.
+
+[POLAR](https://archive.stsci.edu/prepds/polar/), a new High Level Science Product was made available in March by Barros et al. 
+POLAR consists of K2 light curves derived from photometric apertures optimized for bright stars. The light curves have
+been detrended for position-dependent systematic effects, and the group has provided a list of possible planet transits
+selected from this data. 
+
+Finally, in February, [version 2.0 of EVEREST](https://archive.stsci.edu/prepds/everest/) was released, including Campaigns 0 
+through 8. The update now, detailed in Luger et al. (2017), includes light curves for short cadence and saturated targets. 
+Precision has improved over version 1.0, with photometric performance now equivalent to that of the Kepler main mission for stars 
+down to Kp=14.
```

Comment:
Maybe add a paragraph at the end explaining that more detailed documentation is available on each of the HLSP's pages at MAST?